Classes/Menu_TabMisc.uc

```diff
@@ -305,6 +308,16 @@ defaultproperties
      End Object
      Controls(14)=moCheckBox'3SPNvSoL.Menu_TabMisc.ExtendCheck'
 
+     Begin Object Class=moCheckBox Name=WidescreenFixCheck
+         Caption="Enable Widescreen fixes."
+         OnCreateComponent=WidescreenFixCheck.InternalOnCreateComponent
+         WinTop=0.010000
+         WinLeft=0.100000
+         WinWidth=0.800000
+         OnChange=Menu_TabMisc.OnChange
+     End Object
+     Controls(26)=moCheckBox'3SPNvSoL.Menu_TabMisc.WidescreenFixCheck'
```

Comment:
This looks awful, I have no idea how to add a checkbox

Reply:
There is actually a built-in GUI designer.  In User.ini, set bModAuthor=true
```
[XInterface.GUIController]
bModAuthor=false
```
some instructions here
https://wiki.beyondunreal.com/Legacy:Automated_Component

when bModAuthor is true, there are some hotkeys you can press that lets you move controls around in-game, add new controls, etc.  Then it will copy the result to clipboard and you can paste them into your .uc file.  

Whenever you see 'Begin Object, End Object' in the defaultproperties, odds are there is a designer thing somewhere that made that.  There is a similar designer for particle systems in the editor.

I can't use them though.  I find trial and error easier than using those designers.